cloudFerry.py

```diff
@@ -202,5 +262,33 @@ def create_filters(name_config=None, filter_folder=DEFAULT_FILTERS_FILES,
     make_filters.make(filter_folder, images_date)
 
 
+def init(args):
```

Comment:
Please consider detecting following scenario: user runs `cloudferry migrate` *without* running `cloudferry init` first. Warning message should be displayed in that case.